### PR TITLE
Add Contact Us page

### PR DESCRIPTION
**Pull request by ryoarmanda**

### CODE REVIEW

File: src/pages/contact.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import { NextPage } from "next";
+import Link from "next/link";
+
+import { Instagram, Facebook } from "react-feather";
+
+import ContactUsContent from "src/pageContent/ContactUs";
+import Page from "src/components/Page";
+import Text from "src/components/Text";
+
+const Home: NextPage = () => {
```
[the remark this answers is not in the document]
Comment (ryoarmanda) [reply src/pages/contact.tsx:11]: Haha didn't notice this. Will change.

File: src/components/Banner/index.tsx [hunk 2]
```diff
@@ -40,9 +42,12 @@ const Banner: React.FC<OwnProps> = ({
...
         <div className={`bg-secondary w-screen py-10`}>
-          <p className={`text-lg text-center max-w-4xl mx-auto`}>
-            {description}
-          </p>
+          {description && (
+            <p className={`text-lg text-center max-w-4xl mx-auto`}>
+              {description}
+            </p>
+          )}
+          {subBannerContent && subBannerContent()}
```
[the remark this answers is not in the document]
Comment (ryoarmanda) [reply src/components/Banner/index.tsx:50]: Good suggestion, however the naming gets a bit long. How about we drop the `banner`, make it a `renderSubcontent` instead?

File: src/pageContent/ContactUs/index.tsx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    location.href =
+      `mailto:ryoarmanda@u.nus.edu` +
+      `?subject=${encodeURIComponent(subject)}` +
+      `&body=${encodeURIComponent(body)}`;
+  };
+
+  return (
+    <>
+      <div className={S.ContactUsSection}>
+        <div className={`col-start-5 col-span-4 row-start-1 row-span-1`}>
```
[the remark this answers is not in the document]
Comment (ryoarmanda) [reply src/pageContent/ContactUs/index.tsx:49]: I see. I'll try to work it with flexbox.

**Review by jamesyaputra — APPROVED** [review]
LGTM!

**Pull request merged**